src/index.js

```diff
@@ -2,34 +2,10 @@ import React from 'react';
 import 'react-app-polyfill/ie11'; // For IE 11 support
 import ReactDOM from 'react-dom';
 import { OverridableContext } from 'react-overridable';
-import { Link } from 'react-router-dom';
 import 'semantic-ui-less/semantic.less';
-import { Item } from 'semantic-ui-react';
 import { InvenioILSApp } from './lib';
 
-const CustomHome = ({ ...props }) => {
-  return <>And this is custom home</>;
-};
-
-const CustomCover = ({ size, url, isRestricted, asItem, linkTo }) => {
-  const Cmp = asItem ? Item.Image : Image;
-  const link = linkTo ? { as: Link, to: linkTo } : {};
-  return (
-    <Cmp
-      centered
-      disabled={isRestricted}
-      {...link}
-      onError={e => (e.target.style.display = 'none')}
-      src={url}
-      size="tiny"
-    />
-  );
-};
-
-const overriddenCmps = {
-  // 'Home.layout': CustomHome,
-  // 'LiteratureCover.layout': CustomCover,
-};
+const overriddenCmps = {};
```

Comment:
I believe you have removed this because of the warnings in console, but it is handy to test if overridable still works and as example.
What about if you comment it out instead?